cloudbuild-cache.yaml

```diff
@@ -10,17 +10,17 @@ steps:
     path: /root/.ssh
   id: 'untar-mod-cache'
 
-- name: 'golang:1.18.2'
+- name: 'golang:1.20.5'
```

Comment:
Why do we use this version of Go?

In the cloud-builders version we use, https://github.com/solo-io/cloud-builders/blob/v0.6.2/go-mod-make/cloudbuild.yaml#L6, I see `1.20.1`